shared-libs/transitions/src/schedule/due_tasks.js

```diff
@@ -1,7 +1,6 @@
 // TODO: this doesn't need to exist. We can just work this out dynamically when
 //       gateway queries, it's not slow or complicated.
 const async = require('async'),
-  _ = require('underscore'),
```

Comment:
What's the reasoning behind moving the schedule code into the transitions shared lib? AFAICT it's only used by sentinel.

Reply:
It's funny you should ask! 
It's because _someone_ wrote this two years ago: 
https://github.com/medic/medic/blame/8c25c15e6fec268b272d5fb9b361bd119864a4e6/sentinel/src/schedule/due_tasks.js#L1

Reply:
@dianabarsan Could you please raise an issue so this actually gets fixed? I'm happy to do it but you're the sentinel expert now!

Reply:
sure!